x-pack/plugins/security_solution/public/resolver/store/reducer.ts

```diff
@@ -18,7 +18,13 @@ const uiReducer: Reducer<ResolverUIState, ResolverAction> = (
   },
   action
 ) => {
-  if (action.type === 'userFocusedOnResolverNode') {
+  if (action.type === 'serverReturnedResolverData') {
+    const next: ResolverUIState = {
+      ...state,
+      selectedNode: action.payload.result.entityID,
```

Comment:
👍

Reply:
:red_circle: I think you also want here:

> ariaActiveDescendant: action.payload.result.entityID

which sets it as both "current" and "selected"